### PR TITLE
Build MiniStdLib and deps in aarch64 builds

### PR DESCRIPTION
**Pull request by nico**
This pulls the first few parts of AK and Kernel headers into the aarch64 build :^)

### CODE REVIEW

**Comment by nico** [comment]
https://dev.azure.com/SerenityOS/SerenityOS/_build/results?buildId=1495&view=logs&j=2ebe0bba-9b6a-54a9-e644-67b37013ad1f&t=db9c34f3-6624-5520-9c97-fca3483150ca&l=416 =>

```
 FAIL  builtins/WeakSet/WeakSet.prototype.add.js
     ❌ Suite:  <top-level>
```

Looks unrelated.

**Comment by ADKaster** [comment]
That failure looks to be the same as the one we discussed here with no resolution
https://discordapp.com/channels/830522505605283862/851522357734408232/880371782153039882

Basically, the test is trying to make sure that the weak set doesn't hold a strong reference to contained objects. Which ends up making it very sensitive to when the normal GC-ing happens. Happens that this test expects the manual gc invocation to cleanup the weak set, but it doesn't, presumably because something on the stack looked like a pointer to that object.

File: Kernel/MiniStdLib.cpp [hunk 1]
```diff
@@ -10,31 +10,38 @@ extern "C" {
...
         n -= size_ts * sizeof(size_t);
         if (n == 0)
             return dest_ptr;
     }
     asm volatile(
         "rep movsb\n" ::"S"(src), "D"(dest), "c"(n)
         : "memory");
+#else
+    u8* pd = (u8*)dest_ptr;
+    const u8* ps = (const u8*)src_ptr;
```
Comment (linusg) [inline Kernel/MiniStdLib.cpp:41]: east const pls

```suggestion
    u8 const* ps = (u8 const*)src_ptr;
```

**Review by linusg — CHANGES_REQUESTED** [review]

File: Kernel/MiniStdLib.cpp (hunk 1 above)
Comment (nico) [reply Kernel/MiniStdLib.cpp:41]: The whole file uses west const. Is it really policy to make files self-inconsistent?

Comment (linusg) [reply Kernel/MiniStdLib.cpp:41]: yes, current policy is to make all new code use east const (and optionally update stuff around it, e.g. when changing a function signature), event if other things in that same file still use west const. One day the [clang-format folks will finish their discussion](https://reviews.llvm.org/D69764) and we can update all the remaining uses of west const in one go :)

Comment (nico) [reply Kernel/MiniStdLib.cpp]: Done. But if we're going to reformat everything in one fell swoop eventually, is it really worth to:
- modify locally copied-code from west-const to east-const
- spend review time on this

?

**Review by linusg — APPROVED** [review]

**Pull request merged**